src/components/query/PropertyQueryEditor.tsx

```diff
@@ -338,7 +342,7 @@ export class PropertyQueryEditor extends PureComponent<Props, State> {
 
   render() {
     const { query, datasource } = this.props;
-    const { loading, asset, assets } = this.state;
+    const { loading, assets, properties: stateProps } = this.state;
```

Comment:
```suggestion
    const { loading, assets, properties } = this.state;
```

I think since props has an existing meaning in react, it's easy to be confused as to what this means.